src/createServer.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { resetUsers } = require('./services/userService');
+const { resetExpenses } = require('./services/expensesService');
+const { userRouter } = require('./routes/usersRoute');
+const { expenseRouter } = require('./routes/expenseRoute');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  resetUsers();
+  resetExpenses();
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expenseRouter);
```

Comment:
You're using `express.json()` middleware for each route separately. This is not wrong, but it's more common and efficient to use it once for the whole application. This way, you ensure that every incoming request body is parsed as JSON, no matter which route it hits. You can do this by adding `app.use(express.json())` before the routes.